src/Logging/InitializationLogger.cs

```diff
@@ -56,6 +56,7 @@ public static void LogInitializationSucceed(string serviceNameForLogging, string
 		/// <param name="serviceNameForLogging">Service name for logging</param>
 		/// <param name="ex">Exception to log</param>
 		/// <param name="message">Message to log</param>
+		[Obsolete("InitializationLogger using OmexLogger is obsolete and is pending for removal by 1 July 2024.", DiagnosticId = "OMEX188")]
```

Comment:
Is it not enough to mark the class as obsolete?

Reply:
removed